src/config/ConfigMetadata.ts

```diff
@@ -245,6 +245,14 @@ export const ConfigMetadata: {[key: string]: { type: string, defaultValue?: any
         type: 'boolean',
         defaultValue: false,
     },
+    [ConfigNames.THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBLISH_BODY_MASK]: {
+        type: 'boolean',
+        defaultValue: false,
+    },
+    [ConfigNames.THUNDRA_TRACE_INTEGRATIONS_GOOGLE_SUBSCRIPTION_BODY_MASK]: {
```

Comment:
Maybe can have a single config `THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBSUB_MESSAGE_MASK` for both of publish and subscribe